argocd-image-updater/Dockerfile

```diff
@@ -10,15 +10,15 @@ WORKDIR /work/argocd-image-updater
 RUN mkdir -p dist && \
     make controller
 
-FROM quay.io/cybozu/ubuntu:22.04
+FROM ghcr.io/cybozu/ubuntu:22.04
 LABEL org.opencontainers.image.source="https://github.com/cybozu/neco-containers"
 
 RUN groupadd -g 10000 argocd && \
     useradd -r -u 10000 -g argocd argocd && \
     mkdir -p /home/argocd && \
     chown argocd:argocd /home/argocd && \
     apt-get update && \
-    apt-get install -y --no-install-recommends git openssh-client python3 python3-pip && \
+    apt-get install -y --no-install-recommends git openssh-client python3 python3-pip tini gpg gpg-agent && \
     pip3 install --upgrade pip && \
     pip3 install awscli && \
     apt-get clean && \
```

Comment:
We can omit explicit `apt-get clean`.

https://docs.docker.com/build/building/best-practices/
> Official Debian and Ubuntu images [automatically run apt-get clean](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105), so explicit invocation is not required.